build.gradle

```diff
@@ -49,6 +51,17 @@ sonarqube {
     }
 }
 
+jacoco {
+    toolVersion = "0.8.7"
+}
+
+jacocoTestReport.dependsOn(test)
```

Comment:
this could've been added to the jacocoTestReport task below

Reply:
Okay